### ISSUE TITLE
MultiModalBatchNorm

### ISSUE THREAD

**Issue by mahfujau** [mancinimassimiliano/latent_domains_DA#1]
Can you provide your "MultiModalBatchNorm" layer please? I am getting the following error: 

0710 03:13:25.977479  9991 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: MultiModalBatchNorm (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EntropyLoss, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)

**Comment by mancinimassimiliano** [comment]
Thank you for pointing out the error. I have added the missing files. Could you check if everything is okay now?
